File: examples/java-ddd/domain/src/test/java/org/myddd/java/domain/UserTest.java
package org.myddd.java.domain;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import org.myddd.java.AbstractTest;

import javax.transaction.Transactional;

public class UserTest extends AbstractTest {

    @Test
    @Transactional
    void createUser(){
       var user = randomUser().createUser();
        Assertions.assertNotNull(user);
    }
}

File: examples/kotlin-ddd/domain/src/test/kotlin/org/myddd/kotlin/domain/UserTest.kt
package org.myddd.kotlin.domain

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import org.myddd.kotlin.AbstractTest
import jakarta.transaction.Transactional

class UserTest:AbstractTest() {

    @Test
    @Transactional
    fun createUser(){
        val randomUser = randomUser()
        Assertions.assertNotNull(randomUser.createUser())
    }
}